Shared/configurations/ConfigurationController.cpp

```diff
@@ -64,26 +79,183 @@ void ConfigurationController::select(int index)
   emit configurationsChanged();
 }
 
+bool removeDownloadedFile(const QString& downloadFilePath)
+{
+  // remove any remaining progress if request was aborted or after the zip has been extracted
+  QFile f{downloadFilePath};
+  if (f.exists())
+    return f.remove();
+
+  return false;
```

Comment:
Should this return true? If there was no file to remove, I suppose it successfully removed nothing in that case.